sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -47,8 +47,7 @@ export const FastFrameTemplate = html<FastFrame>`
                         <h1><span class="content-heading-highlight">Fast Frame</span> Colors</h4>
                         <h2>Pre-existing color you can customize</h2>
                         <p>
-                            Ultrices nibh nunc vestibulum fames. At lacus nunc lacus eget
-                            neque.
+                            Color recipes are the core of the how color is applied in FAST. There are two core color palettes that are exposed by the design system that are used to derive color for the recipes; the neutral palette and the accent palette. Both can be customized to fit your needs.
```

Comment:
I feel like this could be a little more succinct:
```suggestion
                            Color is applied by using color recipes which require two color palettes, neutral and accent, applied to the design system. These palettes are customizable which allows for a wide range of styles.
```